slash/general.py

```diff
@@ -32,6 +32,57 @@ async def github(self, interaction: Interaction):
             embed=embed, view=Link("https://github.com/codeman1o1/TIJK-Bot")
         )
 
+    @slash(guild_ids=SLASH_GUILDS)
+    async def avatar(
+        self,
+        interaction: Interaction,
+        user=SlashOption(
+            description="The user of which you wish to download their pfp",
+            required=True,
+        ),
+    ):
+        """Get the avatar of you or someone else"""
+        user = user or interaction.author
+
+        avatars_list = []
+
+        def target_avatar_formats(self):
```

Comment:
Please rename the `self` parameter to something more describing

Reply:
done